cicd/gitops/configuring-argo-cd-rbac.adoc

```diff
@@ -0,0 +1,13 @@
+:_content-type: ASSEMBLY
+[id="configuring-argo-cd-rbac"]
+= Configuring Argo CD RBAC
+include::_attributes/common-attributes.adoc[]
+:context: configuring-argo-cd-rbac
+
+toc::[]
+
+[role="_abstract"]
+By default, any user logged into Argo CD using RHSSO will be a read-only user. This guide explains how to manage the user level access.
```

Comment:
To avoid future tense and to make it more user focused, what do you think of the following rephrase?

`By default, if you are logged into Argo CD using RHSSO, you are a read-only user.` 

For the second sentence of the abstract, avoid self referential language, like `this guide explains` (IBM SG 22)

What do you think of this rewrite for the abstract? 

`By default, if you are logged into Argo CD using RHSSO, you are a read-only user. You can change and manage the user level access.`